core/domain/services/graphql/data/mutations/TicketMutation.php

```diff
@@ -67,4 +71,27 @@ public static function setRelatedDatetimes($entity, array $datetimes)
             }
         }
     }
+
+    /**
+     * Sets the related prices for the given ticket.
+     *
+     * @param EE_Ticket $entity The Ticket instance.
+     * @param array     $prices Array of entity IDs to relate.
+     */
+    public static function setRelatedPrices($entity, array $prices)
+    {
+        $relationName = 'Price';
+        // Remove all the existing related entities
+        $entity->_remove_relations($relationName);
```

Comment:
this does not appear to be a very efficient way to go about doing this, since it would potentially have to run multiple queries to do so (I'm guessing the underlying deletions running in `EEM_Base` are happening in a loop) so it might be far more efficient to query for existing relations, then find the difference between the incoming price entities, and then only add and delete records for the actual changes and leave everything else alone.

that said, let's not worry about that now, but can you plz add a `todo` notice here for making that change?

Reply:
Sure, I did look for a single query solutions but the model system does not seem to have one.

Reply:
right... can you plz add a todo note to this though?